vue.config.js

```diff
@@ -3,7 +3,6 @@ module.exports = {
     devtool: 'source-map'
   },
   transpileDependencies: [
-    'vue-plugin-helper-decorator',
```

Comment:
Not sure if I need this or not.

Reply:
Seems to work with or without this but I reverted this change so it's one less thing different.